docs/guides/templates/profile.adoc

```diff
@@ -0,0 +1,11 @@
+<#macro ifProduct>
+ifeval::["{profile}"=="product"]
+<#nested>
+endif::[]
+</#macro>
+
+<#macro ifCommunity>
+ifeval::["{profile}"=="community"]
+<#nested>
+endif::[]
+</#macro>
```

Comment:
Such changes will be probably necessary to add even for the `docs/documentation`, but I don't want to provide it there if there's no use for it ATM.